### ISSUE TITLE
 Remove validation of `utm_campaign` for RTAMO

### ISSUE THREAD

**Issue by jvillalobos** [mozilla/bedrock#10173]
## Description

Currently, Return to AMO (RTAMO) URLs have these params:

`https://www.mozilla.org/firefox/new/?utm_source=addons.mozilla.org&utm_medium=referral&utm_campaign=non-fx-button&utm_content=rta%3AamlkMS1Nbm54Y3hpc0JQblNYUUBqZXRwYWNr`

The last param (utm_content) identifies this as a RTAMO URL and the add-on that will be shown in the first run UX instead of the regular onboarding flow. The other 3 params (utm_source, utm_medium, utm_campaign) have fixed values and [are also validated](https://github.com/mozilla/bedrock/blob/e6da436be7480263be54a72673159a489b23ee7e/media/js/base/stub-attribution-macos.js#L77). If they don't match, the RTAMO attribution doesn't happen.

In order to [experiment with the Download Firefox feature](https://github.com/mozilla/addons-frontend/issues/10218) on AMO and have better attribution tracking, we want to make the `utm_campaign` value variable, so validating it wouldn't be as straightforward. Given that this param validation appears to just be a sanity check, it should be sufficient to stop checking `utm_campaign` and just rely on the other two values.

[There's an equivalent bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1707038) filed for the validation on the Firefox side.

---

#### Success Criteria
- [ ] The new `utm_campaign` scheme doesn't break RTAMO attribution.

/cc @muffinresearch @willdurand

**Comment by alexgibson** [comment]
We can definitely update this for you, but as far as I know the script you link to has never been enabled (see https://github.com/mozilla/bedrock/issues/9346), so it shouldn't be blocking you (if I understand correctly)

**Comment by jvillalobos** [comment]
Thanks. If the script is unused and `utm_campaign` isn't being validated anywhere else, then I think we're good here and we can close this. We can verify that this works once the Firefox patch lands.

**Comment by jvillalobos** [comment]
On second though, the validation code should be updated if there's an expectation that it will be revived once macOS attribution is worked on again (I definitely want RTAMO to work on macOS, but haven't looked into the effort required).

**Comment by alexgibson** [comment]
Closing this as wontfix, as the aforementioned script is being removed. If we ever revisit macOS  attribution again, the code would likely need revisiting / QA'ing again anyway.
